aspnetcore/signalr/get-started.md

```diff
@@ -31,6 +31,10 @@ This tutorial demonstrates the following SignalR development tasks:
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/get-started/sample/) ([how to download](xref:tutorials/index#how-to-download-a-sample))
 
+> [!NOTE]
+> The sample code in this article uses modern ECMAScript 6 features that aren't supported in Internet Explorer 11.
+> To convert the sample for environments that do not support ECMAScript 6, such as Internet Explorer 11, use a transpiler such as Babel.
```

Comment:
Hrm... I did intend for the link to stay in... We should probably put it back. We can just link to the main Babel page: http://babeljs.io/

Reply:
My bad I thought I kept it.